diff -ruN cloudevents-sdk_original/Android.bp cloudevents-sdk/Android.bp
--- cloudevents-sdk_original/Android.bp	2024-02-05 17:23:15.545487000 -0500
+++ cloudevents-sdk/Android.bp	2024-02-05 17:24:28.216983000 -0500
@@ -2,42 +2,6 @@
 // Do not modify this file as changes will be overridden on upgrade.
 
 // TODO: Add license.
-rust_test {
-name: "cloudevents-sdk_test_src_lib",
-host_supported: true,
-crate_name: "cloudevents",
-cargo_env_compat: true,
-cargo_pkg_version: "0.7.0",
-srcs: ["src/lib.rs"],
-test_suites: ["general-tests"],
-auto_gen_config: true,
-test_options: {
-unit_test: true,
-},
-edition: "2018",
-cfgs: ["uuid_unstable"],
-rustlibs: ["libactix_rt", "libbase64_rust", "libbitflags", "libchrono", "libclaim", "libhostname", "libmime", "libmockito", "librmp_serde", "libserde", "libserde_json", "libserde_yaml", "libsnafu", "libtokio", "libtower", "liburl", "libuuid", "libversion_sync"],
-proc_macros: ["libdelegate_attr", "librstest"],
-}
-
-rust_test {
-name: "cloudevents-sdk_test_tests_version_number",
-host_supported: true,
-crate_name: "version_number",
-cargo_env_compat: true,
-cargo_pkg_version: "0.7.0",
-srcs: ["tests/version_number.rs"],
-test_suites: ["general-tests"],
-auto_gen_config: true,
-test_options: {
-unit_test: true,
-},
-edition: "2018",
-cfgs: ["uuid_unstable"],
-rustlibs: ["libactix_rt", "libbase64_rust", "libbitflags", "libchrono", "libclaim", "libcloudevents", "libhostname", "libmime", "libmockito", "librmp_serde", "libserde", "libserde_json", "libserde_yaml", "libsnafu", "libtokio", "libtower", "liburl", "libuuid", "libversion_sync"],
-proc_macros: ["libdelegate_attr", "librstest"],
-}
-
 rust_library {
 name: "libcloudevents",
 host_supported: true,
diff -ruN cloudevents-sdk_original/cargo_embargo.json cloudevents-sdk/cargo_embargo.json
--- cloudevents-sdk_original/cargo_embargo.json	2024-02-05 17:22:50.833658000 -0500
+++ cloudevents-sdk/cargo_embargo.json	2024-02-05 17:24:13.917082000 -0500
@@ -1,3 +1,3 @@
 {
-  "tests": true
+  "tests": false
 }
